tests/lib/rules/await-fire-event.test.ts

```diff
@@ -159,6 +167,26 @@ ruleTester.run(RULE_NAME, rule, {
           ],
         } as const)
     ),
+    ...COMMON_FIRE_EVENT_METHODS.map(
+      (fireEventMethod) =>
+        ({
+          code: `
+      import { fireEvent } from '@marko/testing-library'
+      test('unhandled promise from fire event method is invalid', async () => {
+        fireEvent.${fireEventMethod}(getByLabelText('username'))
+      })
+      `,
+          errors: [
+            {
+              line: 4,
+              column: 9,
+              endColumn: 19 + fireEventMethod.length,
+              messageId: 'awaitFireEvent',
+              data: { name: fireEventMethod },
+            },
+          ],
+        } as const)
+    ),
```

Comment:
I followed what was suggested by @Belco90 earlier. This is actually increasing the scope of PR - there are 2 options to move forward, either the three of us can split the tests among us and update it to support looping over the supported frameworks (_better option_) OR keep the draft PR open and I can send updates as and when I get time to rewrite the test cases. Let me know.

Reply:
I'd say it's out of the scope too. Let's keep it simple and just add some extra tests for Marko to make sure the specific rules are reporting from `@marko/testing-library` correctly.

Then we can rework the tests in a later PR to check all testing frameworks etc.

Reply:
I wasn't saying we should include _all_ testing frameworks now, only the ones we currently have + `@marko/testing-utils`

Reply:
Unless I am missing something, that will still require changes to _all_ tests which would take time.

Reply:
I guess what @MichaelDeBoey means is making the new tests you added to work for `TESTING_FRAMEWORKS` rather than just Marko? If so, I'd say again that's out of the scope. @PrashantAshok already added tests to cover Marko, we should take care of adding remaining tests for other tests frameworks in a different PR.

Reply:
What I meant was: you look at the test you copied & instead of copying it and changing the imported package, you map over an array of 2 testing frameworks (the original one + `@marko/testing-library`).

In a later PR we can update that array to include all testing frameworks

Reply:
@MichaelDeBoey - I fully agree with you. But I personally feel, scope of a PR should be 1 thing and not mix things. The change you are suggesting is not just creating an array but would still require code changes in rest of file as well. 

But if you think we should handle this in this PR - then let's create common branch out of this and we all can tag-team the changes _OR_ keep this PR open _(not ideal)_ and I can push as and when I get time _OR_ merge this PR on the basis of just adding `@marko` and create an issue for refactoring, I would be happy to contribute to that.

Reply:
It's literally 2 extra lines per test, so code changes will be far less then when you're copy/pasting all these tests.

Reply:
Ok fine, will make the change. If you were so concerned about code quality, wish you would have done this earlier, rather than wait for adding support for a new framework.

Btw as I mentioned earlier the import does not really matter 
```import {render} from '@testing-library/some-invalid-name``` still make the tests work, so hopefully you are going to look into that.

Reply:
It's only now appearing, as you're now actually copy/pasting the tests, that's why I only say it now.

That we don't have tests for all frameworks we support, is indeed something we should have look into earlier.

Reply:
> It's only now appearing, as you're now actually copy/pasting the tests, that's why I only say it now.
> 
> 
> 
> That we don't have tests for all frameworks we support, is indeed something we should have look into earlier.

This is why is out of the scope. This PR is for including Marko and @PrashantAshok did a good job adding new tests for covering it. How previous frameworks were tested shouldn't be a matter of this PR, unless they were all tested following a pattern not matched by Marko, but that's not the case.

Let's move on and keep it simple so we can get the support for Marko merged, then in a separate issue/PR we decide how to check all the necessary frameworks.

Reply:
Let's indeed agree to disagree to get this awesome PR merged! 👍